### ISSUE TITLE
Language selected for file is reset to none after adding word to user dict

### ISSUE THREAD

**Issue by slodki** [bartosz-antosik/vscode-spellright#42]
Each time I add word to user dictionary language indicated at status bar is reset to [none].
Underlines and suggestion tooltips inside editor, spelling problems in PROBLEMS tab are preserved, but yellow bulb is not shown anymore.

After selecting language again it works... till next word added to dict.

**Comment by bartosz-antosik** [comment]
Could you please name the platform (macOS, Linux)? 

It seems it does not happen on Windows (at least Windows 10).

**Comment by slodki** [comment]
VS Code from Linux Ubuntu zesty binary package:
> Version 1.15.0
> Commit 8b95971d8cccd3afd86b35d4a0e098c189294ff2
> Date 2017-08-09T20:09:15.006Z
> Shell 1.6.6
> Renderer 56.0.2924.87
> Node 7.4.0
> Architecture x64

Your extension installed just moment ago from Marketplace (version 1.1.8 IMHO).

**Comment by bartosz-antosik** [comment]
Thanks for the update.

**Comment by bartosz-antosik** [comment]
Would I be correct if I would rephrase the subject that it does not reset to none but reset to what you have set in configuration? E.g. if you would set in user settings "spellright.language" to "en" (or whatever name of the dictionary you have) then after 1) changing to another language than what you have set above and 2) adding the word to user dictionary the language would get switched back to "en"?

The byproduct of this issue would then be that on Linux the default spelling dictionary taken from OS locales is set to an empty string...

**Comment by bartosz-antosik** [comment]
Should be corrected in version 1.1.12.
